silabs-multiprotocol/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.4.1
+
+- Revert back to Silicon Labs Gecko SDK 4.3.1
```

Comment:
I'd still like to receive feedback regarding stability of this release (with ZGP removed). Maybe we can extend the message?

```suggestion
- Revert back to Silicon Labs Gecko SDK 4.3.1 to maintain compatibility with Home Assistant Core 2023.12.0 and Zigbee2MQTT. If you are running 2024.1.0 with ZHA, you don't have to downgrade.
```

Reply:
Yeah make sense :+1: